momento-sdk/src/intTest/java/momento/sdk/BaseTestClass.java

```diff
@@ -19,8 +19,7 @@ static void beforeAll() {
   }
 
   private static void ensureTestCacheExists() {
-    try (SimpleCacheClient client =
-        SimpleCacheClient.builder(System.getenv("TEST_AUTH_TOKEN"), 10).build()) {
+    try (CacheClient client = CacheClient.builder(System.getenv("TEST_AUTH_TOKEN"), 10).build()) {
```

Comment:
Just FYI - You might need a rebase if I push the Duration changes before this PR is merged!

Reply:
this change is simpler than the duration one, so I think we should merge the duration one first and then rebase this one afterward.